src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -38,7 +37,7 @@ const formatPhone = (phone) => {
   }
 };
 
-const PersonalInfoList = ({ myProf, profile, createSnackbar }) => {
+const PersonalInfoList = ({ myProf, profile, network, createSnackbar }) => {
```

Comment:
```suggestion
const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
```
It might be clearer to name this property in a boolean-ish way rather than just "network". Could you change this in the 3+ places where it appears?

Reply:
Also @EjPlatzer , is it better to use the hook everywhere or is it alright to pass the value from higher level components down.

Reply:
This Evan's response to my concern with the duplicated imports, then he told me the imports are find, but not calling the hooks multiple times. I also checked out other components inside Profile, the Identification also uses network prop.
![image](https://user-images.githubusercontent.com/78691207/196255505-dbf60483-f77e-45f5-98eb-a9ddb4561762.png)

Reply:
Great, that answers that then. Just the change to `isOnline` for my recommendation then.

Reply:
![image](https://user-images.githubusercontent.com/78691207/196303045-b6ddb838-d0ab-4dd8-b670-6f9676d289f1.png)
I don't think that works, because isOnline has already declared as a const.

Reply:
If you delete the line for defining the const after updating the property name, it should work.